olp-cpp-sdk-core/src/http/curl/NetworkCurl.cpp

```diff
@@ -1058,5 +1071,25 @@ void NetworkCurl::Run() {
   OLP_SDK_LOG_DEBUG(kLogTag, "Thread exit, this=" << this);
 }
 
+void NetworkCurl::GetTraficData(CURL* handle, uint64_t& uploadBytes,
+                                uint64_t& downloadBytes) {
+  long headersSize;
+  double lengthDownloaded;
+  if (curl_easy_getinfo(handle, CURLINFO_HEADER_SIZE, &headersSize) ==
+          CURLE_OK &&
+      curl_easy_getinfo(handle, CURLINFO_SIZE_DOWNLOAD, &lengthDownloaded) ==
+          CURLE_OK) {
+    downloadBytes = lengthDownloaded + headersSize;
+    OLP_SDK_LOG_DEBUG(kLogTag, "downloaded bytes " << downloadBytes);
```

Comment:
Log without any additional information will be very hard to read. At least the url or the request id needs to be also there so that we can associate it somehow to an existing request.

Reply:
removed logging